ndc-client/Cargo.toml

```
@@ -11,6 +11,7 @@ rustls = ["reqwest/rustls"]
 
 [dependencies]
 async-trait = "^0.1.74"
+bytes = "1.5.0"
```

Any reason not to use the latest version?

```suggestion
bytes = "1.6.0"
```